demo/radar.html

```diff
@@ -695,27 +670,6 @@
 	});
 });
 
-	// let gapShaderMaterial = shaderMaterial.clone();
```

Comment:
leave this

Reply:
Going to delete and track in this issue: https://app.zenhub.com/workspaces/veritas-5b9aa680dfeccd3433cd6f62/issues/nextdroid/potree/25